windows/cc/WindowWin32.cc

```diff
@@ -74,6 +74,15 @@ void jwm::WindowWin32::setIcon(const std::wstring& iconPath) {
     HICON hicon = (HICON)LoadImage(NULL, iconPath.c_str(), IMAGE_ICON, 0, 0, LR_LOADFROMFILE | LR_DEFAULTSIZE);
     SendMessage(_hWnd, WM_SETICON, ICON_SMALL, (LPARAM)hicon);
 }
+
+void jwm::WindowWin32::setOpacity(int opacity) {
+    JWM_VERBOSE("Set window opacity'" << opacity << "'");
+    LONG previousStyle = GetWindowLong(_hWnd, GWL_EXSTYLE);
+    LONG opacityEnabled = previousStyle |= WS_EX_LAYERED;
+    SetWindowLong(_hWnd, GWL_EXSTYLE, opacityEnabled);
```

Comment:
I am not proficient in WinAPI, but it feels like if opacity is set to 1.0 (fully opaque), we should remove WS_EX_LAYERED instead of adding it. Correct me if I’m wrong.

Reply:
Yes, I think it is better. I will fix it.